core/common/src/main/java/alluxio/wire/MetricValue.java

```diff
@@ -24,17 +24,22 @@
 @NotThreadSafe
 public final class MetricValue {
 
-  private Double mDoubleValue;
-  private Long mLongValue;
+  private Double mDoubleValue = null;
```

Comment:
In Java object fields are initialized to `null` by default, so we don't usually set it explicitly

Reply:
Fixed. Thanks for pointing out that!